packages/tab/index.tsx

```diff
@@ -203,7 +206,9 @@ export default class Tab extends React.Component<TabProps, TabState> {
     const {tabIndex, ['aria-selected']: ariaSelected} = this.state;
 
     return (
-      <button
+      // https://github.com/Microsoft/TypeScript/issues/28892
+      // @ts-ignore
+      <Tag
```

Comment:
can you also add a unit test to ensure that this renders as an anchor with an href?